src/state/slices/swappersSlice/selectors.ts

```diff
@@ -101,3 +112,11 @@ export const selectManualReceiveAddressIsValidating = createSelector(
   selectSwappers,
   swappers => swappers.manualReceiveAddressIsValidating,
 )
+
+export const selectSellAmountUsd = createSelector(
+  selectSellAmountCryptoPrecision,
+  selectSellAssetUsdRate,
+  (sellAmountCryptoPrecision, sellAssetUsdRate) => {
+    return bn(sellAmountCryptoPrecision).times(sellAssetUsdRate).toFixed()
+  },
+)
```

Comment:
these selectors were added because we need to track the inputted sell amounts, not the quote sell amounts since errored quotes are not guaranteed to have any data to pull the info from

Reply:
i.e the t`radeQuoteSlice` relies on and `activeQuote` which may not exist in the event all quotes errored